mandatory/1-fix-functions.js

```diff
@@ -10,31 +10,29 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function mood(feelings) {
+  if ((feelings != false)) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
```

Comment:
nice - you don't need to extra set of parenthesis, but this is a good example of how you can remove the `isHappy` variable and just use a boolean argument instead - good to see :P